IFS08_Iscritti-per-fasce-ISEE.yaml

```
@@ -203,13 +186,13 @@ components:
         maxValue:
           type: integer
           format: int32
-          description: 'Valore massimo dell''ISEE per la fascia corrispondente. Se la fascia non ha un valore massimo definito, viene impostato su "null"'
+          description: Valore massimo dell''ISEE per la fascia corrispondente. Se la fascia non ha un valore massimo definito, viene impostato su "null"
```

```suggestion
          description: Valore massimo dell''ISEE per la fascia corrispondente. Se la fascia non ha un valore massimo maggiore di 10, viene impostato su "null"
```
@mfortini dovremmo capire questo valore massimo per la fascia poi ti spiego